src/options.cpp

```diff
@@ -2169,7 +2169,7 @@ void options_manager::add_options_world_default()
 
     add( "ITEM_SPAWNRATE", "world_default", to_translation( "Item spawn scaling factor" ),
          to_translation( "A scaling factor that determines density of item spawns.  A higher number means more items." ),
-         0.01, 10.0, 1.0, 0.01
+         0.00, 10.0, 1.0, 0.00
```

Comment:
If I'm not mistaken, the last number here is a step.
```suggestion
         0.00, 10.0, 1.0, 0.01
```

Reply:
This is why I need to test my stuff! Thanks for catching that.